mne/dipole.py

```diff
@@ -346,6 +346,14 @@ class DipoleFixed(object):
     verbose : bool, str, int, or None
         If not None, override default verbose level (see :func:`mne.verbose`
         and :ref:`Logging documentation <tut_logging>` for more).
+    pos : array, shape (n_dipoles, 3)
+        The dipoles positions (m) in head coordinates.
+    amplitude : array, shape (n_dipoles,)
+        The amplitude of the dipoles (Am).
+    ori : array, shape (n_dipoles, 3)
+        The dipole orientations (normalized to unit length).
+    gof : array, shape (n_dipoles,)
+        The goodness of fit.
```

Comment:
Hi @Eric89GXL I think I'll remove all the tests that use the follow attributes: `ori`, `pos`, `gof` and amplitude, because these attributes don't belong to the `DipoleFixed` object.
I'll try to make other tests in `test_rap_music.py` using first, last, times and other attributes of `DipoleFixed` to have some numeric tests, can be?